core/controllers/editor.py

```diff
@@ -137,7 +137,9 @@ class ExplorationHandler(EditorHandler):
                     'type': 'list',
                     'items': {
                         'type': 'object_dict',
-                        'object_class': exp_domain.ExplorationChange
```

Comment:
@lkbhitesh07 I'm assuming earlier we were validating the entire change object for all the cases, right? If yes, please don't remove this, as this is a more exhaustive check than just introducing a check for rules/cust_arg.

Reply:
We are also validating the complete change object in my changes too. So when I return the `ExplorationChange` domain object from the function it calls its validation method directly. I'm just adding some extra validation check on top of it.